exercises/nondet_monad_seq/TODO

```diff
@@ -0,0 +1,19 @@
+-- about this exercise ---------------------------------------------------------
+
+* Add a link to the exercise where the nondeterminism monad is used.
+
+* Should `fix` be part of the constructors?
+  How do we define `nats`?
+
+* Should also test computations that are expected to produce an infinite
+  number of results. E.g., grade `nats`.
+
+* The error messages that are produced when the code is not lazy (too eager)
+  are admittedly cryptic. Could we produce a better explanation?
+
+-- ocaml questions -------------------------------------------------------------
+
+-- learn-ocaml questions -------------------------------------------------------
+
+* Some tests that succeed when run in batch mode
+  fail with `Stack_overflow` when run in Firefox.
```

Comment:
Can you give me your version of Firefox, please?

Reply:
Confirmed a few minutes ago with Firefox 72.0.2. When grading my own solution to the exercise `nondet_monad_seq`, in the second half of Question 4 ("Laziness"), the grader dies with a stack overflow. Yet, on the command line, the grader works. My version of `learn-ocaml` is bd002531a99a0f682e06a64e328b8e9fec8932e4 (October 2019). I just tried again with the latest version (7f3ce568000b042986d735745b27fb5b53a56dfb) and the problem is the same.

Reply:
Thanks.